packages/renderer/src/lib/dashboard/ProviderWarnings.svelte

```diff
@@ -18,7 +18,7 @@ $: {
     {#each providerInfo.warnings as warn}
       <div class="flex-row items-center align-middle mt-0.5">
         <!-- Make line height center-->
-        <span class="ml-1 text-md text-gray-700">⚠</span>
+        <span class="ml-1 text-md text-gray-700">{warn.icon || '⚠'}</span>
```

Comment:
there is no such icon field on the `providerInfo.warnings` fields

Reply:
pretty sure that I added one to the interface, does it need more ?